[2024-01-27T04:43:10.112+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T01:00:00+00:00 [queued]>
[2024-01-27T04:43:10.131+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T01:00:00+00:00 [queued]>
[2024-01-27T04:43:10.132+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-27T04:43:10.198+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_weather_data> on 2024-01-27 01:00:00+00:00
[2024-01-27T04:43:10.213+0000] {standard_task_runner.py:60} INFO - Started process 4327 to run task
[2024-01-27T04:43:10.232+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'get_weather_data', 'scheduled__2024-01-27T01:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/weather_api__dag.py', '--cfg-path', '/tmp/tmpmuy9ior9']
[2024-01-27T04:43:10.234+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask get_weather_data
[2024-01-27T04:43:10.380+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T01:00:00+00:00 [running]> on host d24a8cdd2842
[2024-01-27T04:43:10.536+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='get_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-27T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-27T01:00:00+00:00'
[2024-01-27T04:43:10.540+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-27T04:43:10.550+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=get_weather_data, execution_date=20240127T010000, start_date=20240127T044310, end_date=20240127T044310
[2024-01-27T04:43:10.582+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-27T04:43:10.605+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
